### ISSUE TITLE
Font issue

### ISSUE THREAD

**Issue by lameditationguidee** [GavickPro/Portfolio-Free-WordPress-Theme#60]
I'm using GK Portfolio.
On pages and articles i choose a font for the text. It appear good on my admin panel but when i sign out or i use an other broser it is not the good font ?

**Comment by dziudek** [comment]
Hello,

Please provide an URL to your website - without it we are unable to help you with your issue.

**Comment by lameditationguidee** [comment]
Hello,

Sorry, my url website is : http://lameditationguidee.com <http://lameditationguidee.com/>
Gaël

**Comment by dziudek** [comment]
Most probably you have set a wrong Font name:

```css
body { font-family: arial }
        .site-title { font-family: arial }
```

it should be **Arial** not **arial**

**Comment by lameditationguidee** [comment]
Hi,

Thank you for your answer.

I really appreciate your them that’s why i’m insisting...

I did not set a wrong font. I edited the post, pages and as you will see on the pict below the font name is symbol. But if you check « edition » page on the website it is an arial font ?

One person contact me with another issue with the font as you will see with the second pict. All the website appear like that but this is the link of the post you can see below : http://lameditationguidee.com/la-meditation-est-elle-la-pratique-appropriee-a-sa-recherche/ <http://lameditationguidee.com/la-meditation-est-elle-la-pratique-appropriee-a-sa-recherche/>
I verified on other computers, all Mac, but i haven’t found this issue ?

Regards,

Gaël

**Comment by dziudek** [comment]
Please provide me a screenshot of your "Font options" area in the theme customizer.

**Comment by lameditationguidee** [comment]
Hi,

I’m using your them gk portfolio and i’m really enjoying it.

I had a first font issue. You gave me the solution but the font option is really making things difficult because what i see on wordpress administration window is not always what i get on some computers. It will be more simple to have the possibility to disable this option…

My first issue was to choose a font when i’m creating an article on wordpress. I choosed symbol font on the writing wordpress tool  and on the font option them it was Arial. On certain computers the font who was appearing was a king of Grec symbol ? The issue was solved by having the same font on wordpress writing tool and them option (Arial)...

My new issue is with hyphenation. It’s good on my computer and my ipad. But one of my friend get something different on his computer at home and at work. The words are cutting anywhere as you will see on the pict of the print i’m sending to you. The print is exactly the same as he’s seing on his both computers.

I tryed to use a plugin for hyphenation but it doesn’t change anything because them font option is disable it. I thought maybe it’s because i justified blocks on the left and right and i modified it to be only justified on the left but it’s not working also...

My blog : http://lameditationguidee.com <http://lameditationguidee.com/>

The article who was taken the print i’m sending to you : http://lameditationguidee.com/les-postures-au-sol-detaillees/ <http://lameditationguidee.com/les-postures-au-sol-detaillees/>

Thank you by advance to help me to find a solution…

Regards,

Gaël
